src/app/wishlists/wishlist-file.test.ts

```diff
@@ -3,11 +3,11 @@ import { toWishList } from './wishlist-file';
 
 const cases: [wishlist: string, result: WishListRoll][] = [
   [
-    'dimwishlist:item=-69420&perks=3643424744,2896038713#notes:Over, FocuFury',
+    'dimwishlist:item=-69420&perks=2682205016,2402480669#notes:Enh Over, Enh FocuFury',
     {
       itemHash: -69420,
-      recommendedPerks: new Set([3643424744, 2896038713]),
-      notes: 'Over, FocuFury',
+      recommendedPerks: new Set([2682205016, 2402480669]),
+      notes: 'Enh Over, Enh FocuFury',
```

Comment:
Why change this case instead of adding a new one?

Reply:
The expected results of this original test case have changed - I'll add some more documentation.